librz/include/rz_type.h

```diff
@@ -24,11 +24,15 @@ typedef struct rz_type_target_t {
 	bool big_endian;
 } RzTypeTarget;
 
+typedef struct rz_type_parser_t RzTypeParser;
+
 typedef struct rz_type_db_t {
 	void *user;
-	Sdb *sdb_types;
-	Sdb *formats; // for `pf` formats
+	HtPP *types; // A hashtable of RzBaseType
```

Comment:
Better like this imo:
```suggestion
	HtPP /* <char *, RzBaseType *> */ *types; ///< name -> base type
```